webapp/src/components/MarketplacePage/Publication/Publication.js

```diff
@@ -0,0 +1,70 @@
+import React from 'react'
+import { Header, Card, Button, Icon } from 'semantic-ui-react'
+import { Link } from 'react-router-dom'
+import { publicationType } from 'components/types'
+import { txUtils } from 'decentraland-commons'
+import EtherscanLink from 'components/EtherscanLink'
+import numeral from 'numeral'
+import moment from 'moment'
+import './Publication.css'
+
+export default class Publication extends React.PureComponent {
+  static propTypes = {
+    publication: publicationType
+  }
+  render() {
+    const { publication } = this.props
+    let iconName = 'check circle outline'
+    let iconTooltip = 'Transaction confirmed'
+    if (publication.tx_status === txUtils.TRANSACTION_STATUS.pending) {
+      iconName = 'warning sign'
+      iconTooltip = 'Transaction pending'
+    } else if (publication.tx_status === txUtils.TRANSACTION_STATUS.failed) {
+      iconName = 'remove circle outline'
+      iconTooltip = 'Transaction failed'
+    }
+    const isExpired = publication.expires_at < Date.now()
+    return (
+      <Card className="Publication">
+        <Card.Content>
+          <Link to={`/${publication.x}/${publication.y}`}>
+            <Icon name="map" />
+            {publication.x},{publication.y}
+          </Link>
+          <Card.Meta>
+            {isExpired
+              ? `Expired`
+              : `Expires ${moment(publication.expires_at).fromNow()}`}
+          </Card.Meta>
+          <span
+            data-balloon-pos="up"
+            data-balloon={iconTooltip}
+            className="tx-icon"
+          >
+            <EtherscanLink txHash={publication.tx_hash}>
+              <Icon name={iconName} className={publication.tx_status} />
+            </EtherscanLink>
+          </span>
+        </Card.Content>
+        <Card.Content extra>
+          <span className="footer">
+            <Header size="medium" floated="left" className="price">
+              {numeral(publication.price).format('0,0.00')} MANA
```

Comment:
I have two pet peeves here:

### 1 

I think it's better to show the entire number and use text-overflow with a tooltip if it's too long, to avoid confusion.
Furthermore, I think it's distracting to _always_ show decimals, like in `23.000,00 MANA`. I like `23.000 MANA` more

### 2

I'm not agains using `numeral` here but I think we'll need a better abstraction, as we are hardcoding US-based formatting. The rest of the codebase is using `toLocaleString` so is not doing a good job of fixing decimals, but at least we follow the browser locale.

For example we could have a `numberFormatter.js` (or just `numeral`) on `./lib` with one or more `format` methods so we can abstract number formatting in a single place.

---

What do you think?